requests/differential_getrawdiff.go

```diff
@@ -0,0 +1,9 @@
+package requests
+
+import "github.com/thought-machine/gonduit/responses"
+
+// DifferentialGetRawDiffRequest represents a request to differential.getrawdiff.
+type DifferentialGetRawDiffRequest struct {
+	DiffID responses.SearchData `json:"diffID"`
```

Comment:
this doesn't feel like it should be a responses.SearchData I think its just a string right?

Reply:
Ah you're right! Think I was messing around with something else and got confused here